doc/api/process.md

```diff
@@ -288,10 +288,10 @@ needed.
 <!-- YAML
 added: v1.4.1
 changes:
-  - version: v7.0.0
+  * version: v7.0.0
```

Comment:
I'm going to stop here, but this is not the last instance in this change set of altering YAML embedded in an HTML comment rather than a markdown unordered list.